build.gradle

```diff
@@ -1104,6 +1104,8 @@ project(":iceberg-spark3-extensions") {
     testCompile project(path: ':iceberg-spark', configuration: 'testArtifacts')
     testCompile project(path: ':iceberg-spark3', configuration: 'testArtifacts')
 
+    testCompile "org.apache.avro:avro"
```

Comment:
In the original PR https://github.com/apache/iceberg/pull/3273, this was in `spark/v3.0/build.gradle`. But we're not including the structural refactor in the point release.

Additionally, this was `testImplementation`. Given that the above dependencies were also changed to `testImplementation`, I chose to be consistent and use `testCompile`.